src/examples/zkapps/voting/members.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  Field,
+  SmartContract,
+  state,
+  State,
+  method,
+  DeployArgs,
+  Permissions,
+} from 'snarkyjs';
+
+export class Members extends SmartContract {
```

Comment:
Should we name it `Membership`?

Reply:
I'm fine with changing it to Membership.